hack/update-codegen.sh

```diff
@@ -38,7 +38,7 @@ informergen=$(kube::util::find-binary "informer-gen")
 # that generates the set-gen program.
 #
 
-GROUP_VERSIONS=(${KUBE_AVAILABLE_GROUP_VERSIONS})
+IFS=" " read -r -a GROUP_VERSIONS <<< "${KUBE_AVAILABLE_GROUP_VERSIONS}"
```

Comment:
change reason:
```
In ./hack/update-codegen.sh line 41:
GROUP_VERSIONS=(${KUBE_AVAILABLE_GROUP_VERSIONS})
                ^-- SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
```